deployment-automation-framework/puppet/manifests/site.pp

```diff
@@ -0,0 +1,45 @@
+node 'default' {
```

Comment:
@Suhan - Please run Python code review tool and fix the issues 
some tools are listed in - http://stackoverflow.com/questions/10163307/are-there-tools-available-for-python-source-code-analysis